src/main/resources/avro/bdg.avdl

```diff
@@ -286,6 +299,13 @@ record Fragment {
    */
   array<Sequence> sequences = [];
   array<AlignmentRecord> alignments = [];
+
+
+  /**
+   Platform-specific information about this Fragment.
+   */
+  union { null, Platform } platformType;
+  union { null, IlluminaHTInfo } platformInfo;
```

Comment:
How do you deal with multiple reads that come from different positions in the flow cell?